go.mod

```diff
@@ -4,34 +4,32 @@ go 1.13
 
 require (
 	github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
-	github.com/containers/buildah v1.14.11
-	github.com/containers/common v0.8.4
-	github.com/containers/image/v5 v5.4.3
-	github.com/containers/storage v1.18.2
+	github.com/containers/buildah v1.16.0
+	github.com/containers/common v0.21.0
+	github.com/containers/image/v5 v5.5.2
+	github.com/containers/storage v1.23.3
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/elazarl/goproxy v0.0.0-20191011121108-aa519ddbe484 // indirect
-	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
-	github.com/fsouza/go-dockerclient v1.6.3
-	github.com/googleapis/gnostic v0.3.1 // indirect
-	github.com/opencontainers/runc v1.0.0-rc9
-	github.com/opencontainers/runtime-spec v1.0.1
+	github.com/fsouza/go-dockerclient v1.6.5
+	github.com/opencontainers/runc v1.0.0-rc91.0.20200707015106-819fcc687efb
+	github.com/opencontainers/runtime-spec v1.0.3-0.20200710190001-3e4195d92445
 	github.com/openshift/api v0.0.0-20200326160804-ecb9283fe820
 	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
-	github.com/openshift/imagebuilder v1.1.4
+	github.com/openshift/imagebuilder v1.1.6
 	github.com/openshift/library-go v0.0.0-20200327125526-163b2f0d6264
 	github.com/openshift/source-to-image v1.3.0
 	github.com/pkg/errors v0.9.1
-	github.com/seccomp/containers-golang v0.0.0-20190710130535-8d1ed6a872a5 // indirect
-	github.com/spf13/cobra v0.0.7
+	github.com/sclevine/agouti v3.0.0+incompatible // indirect
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/sys v0.0.0-20200327173247-9dae0f8f5775
-	k8s.io/api v0.18.0
-	k8s.io/apimachinery v0.18.0
-	k8s.io/client-go v0.18.0
-	k8s.io/component-base v0.18.0
+	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
+	k8s.io/api v0.19.3
+	k8s.io/apimachinery v0.19.3
+	k8s.io/client-go v0.19.3
+	k8s.io/component-base v0.19.3
 	k8s.io/klog v1.0.0
-	k8s.io/kubectl v0.18.0
-	k8s.io/kubernetes v1.18.0
+	k8s.io/kubectl v0.19.3
+	k8s.io/kubernetes v1.19.3
```

Comment:
Kube should not be bumped a y version. The correct fix version is 1.18.10 (`0.18.10` for most k8s.io bumps).

See https://github.com/kubernetes/kubernetes/issues/95622 for the list of fix versions upstream.

Reply:
Corrected in current push